genesyscloud/resource_genesyscloud_routing_queue.go

```diff
@@ -1308,6 +1309,16 @@ func buildSdkDirectRouting(d *schema.ResourceData) *platformclientv2.Directrouti
 			UseAgentAddressOutbound: &messageUseAgentAddressOutbound,
 		}
 
+		if backupQueueID == "" || backupQueueID == NullValue {
```

Comment:
I don't think your second condition is needed. `NullValue` is a string with a value of null so `backupQueueID == NullValue` is checking if `backupQueueID == "null"`. I don't know a situation this would be used since if `backup_queue_id` was not supplied `backupQueueID` would be an empty string